### DISCUSSION TITLE
Some suggestions for quantum equipment

### DISCUSSION THREAD

**Discussion by jimahy7687** [TechReborn/TechReborn#2513] [Ideas]
**Is your feature request related to a problem? Please describe.**
1. Quantum shoes should be immune to fall damage
2. Quantum helmet should have night vision

**Comment by ghost** [comment]
My $.02:
I don't think Quantum Boots should provide fall damage immunity for free.  Having the negated fall damage cost a certain amount of energy would be neat though.
Night vision on the helmet sounds great, but should also come at an energy cost, and preferably a key to toggle the functionality.

**Comment by jimahy7687** [comment]
> My $.02:
> I don't think Quantum Boots should provide fall damage immunity for free. Having the negated fall damage cost a certain amount of energy would be neat though.
> Night vision on the helmet sounds great, but should also come at an energy cost, and preferably a key to toggle the functionality.

all cost energy of course

**Comment by WrongWeekend** [comment]
Water breathing on the quantum helmet may also be useful, which would consume additional energy if toggled on and in effect.

**Comment by drcrazy** [comment]
Quantum helmet already provides waterbreathing

**Comment by ghost** [comment]
I think helmet should provide saturation (at the cost of energy of course). I had some problems with it and found it quite irritating when running long distanced with quantum leggins on and after some short period of time it just starts hurting because there are no hunger points left. According to my test in superflat world it takes roughly 55 seconds to get hunger from 20 to 0 when running straight, which is quite fast.

**Comment by WrongWeekend** [comment]
Perhaps a feeding feature could be supplied by a "Nutrient Cell" item produced in the solid canning machine.

**Comment by ghost** [comment]
Totally agree with you
